README.md

```diff
@@ -1,5 +1,15 @@
 Anita & Jeri - Team FedAx
 
+> ## heroku URLs
+
+FedAx API
+- http://fedax.herokuapp.com
+- shipping quote endpoint: GET http://fedax.herokuapp.com/quote
+- shipping finalization endpoint: POST http://fedax.herokuapp.com/ship
+
+CosMade (bEtsy) online retailer (version that works with FedAx API)
+- http://cosmade.herokuapp.com
+
```

Comment:
Good job updating the README. This is an often neglected step in development but a very important one.

Additional information that would have been especially good to include are details about how to use your api. So for instance I have gone to the core site and it tells me what was last shipped. I don't find anything to click on and so I go to the GET endpoint and get this {"message":"Param is missing or the value is empty: origin"}. 

This all makes perfect sense, it is meant to be used with another site which you have put below, which is great.

However, is this meant to be a private API that only CosMade can use? Or is it meant to be a more general API that one could use? If it is meant to be private it may be good to state that explicitly. If it is meant to be public or if your CosMade devs are going to be doing any additional interaction with it, it would be good to have some documentation for how to use this.

I understand that you probably didn't have the time for such concerns so this is in no way a criticism. Simply something that is extremely important to think about as you move forward into your capstone and the rest of your career.